protocols/raft/src/main/java/io/atomix/protocols/raft/roles/LeaderRole.java

```diff
@@ -341,7 +341,9 @@ private boolean initializing() {
 
           return appender.appendEntries(entry.index()).whenComplete((commitIndex, commitError) -> {
             raft.checkThread();
-            raft.getStateMachine().<OperationResult>apply(entry.index());
+            if (isRunning() && commitError == null) {
```

Comment:
Log warning? Debug?

Reply:
I've found this typically means the leader transitioned back to another role before the change could be committed, so other log messages already indicate what happened and the failure can just be ignored

Reply:
Hmm actually should probably still log it

Reply:
At least debug.

Reply:
Actually, no logging is needed here. The _only_ case where this can fail is actually when a leader transitions back to another role, and the failure would result in an `ERROR` response to the reconfiguring node, so logging the error would be redundant.